administrate/app/assets/stylesheets/administrate/components/_search.scss

```diff
@@ -38,6 +53,7 @@
 }
 
 .search__hint {
+  z-index: 1;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Properties should be ordered color, opacity, z-index